src/test/resources/features/pre_and_post_installation_hooks.feature

```diff
@@ -26,12 +26,12 @@ Feature: Hooks
 		Then I see "Pre-installation hook failure"
 		And the exit code is 1
 
-	Scenario: Post-install Hook returns a non-zero code
-		And an "x86_64" machine with "Linux" installed
-		And the system is bootstrapped
-		And the candidate "grails" version "2.1.0" is available for download on "Linux" with architecture "x86_64"
-		And a "pre" install hook is served for "grails" "2.1.0" on "Linux" with architecture "x86_64" that returns successfully
-		And a "post" install hook is served for "grails" "2.1.0" on "Linux" with architecture "x86_64" that returns a failure
-		When I enter "sdk install grails 2.1.0"
-		Then I see "Post-installation hook failure"
-		And the exit code is 1
+#	Scenario: Post-install Hook returns a non-zero code
```

Comment:
I guess we can delete this one completely.

Reply:
On second thought, I think this entire feature file and all it's related step defs can go. I also suggest ripping out the pre-install hooks as part of this. We no longer use those either.